### PR TITLE
reset form on submit feedback

### PR DESCRIPTION
**Pull request by jyo142**
Closes-https://github.com/salemlf/hakubun/issues/205

I commented out the github submitting code for testing this :)

https://github.com/salemlf/hakubun/assets/4103523/8390a6b3-11c5-4be9-bf02-c3f42475624d

### CODE REVIEW

File: src/components/UserFeedbackModal/UserFeedbackModal.tsx [hunk 1]
```diff
@@ -183,16 +188,34 @@ function UserFeedbackModal({ isOpen, setIsOpen }: Props) {
...
+
   const feedbackTypeSelectorRef = useRef<HTMLButtonElement>(null);
-  const [isUsernameIncluded, setIsUsernameIncluded] = useState<boolean>(false);
-  const [isDeviceInfoIncluded, setIsDeviceInfoIncluded] =
-    useState<boolean>(false);
+
   const { isReportSubmitting, createAndPostIssue, setIsReportSubmitting } =
-    useUserFeedbackSubmit(setIsOpen);
-  const [selectedFeedbackType, setSelectedFeedbackType] =
-    useState<FeedbackTypeSelection>(feedbackTypeSelections[0]);
```
Comment (jyo142) [inline src/components/UserFeedbackModal/UserFeedbackModal.tsx:195]: moved these fields above and added default fields via an object

**Comment by jyo142** [comment]
@salemlf please take a look when you have time! Thank you for letting me contribute!

**Comment by salemlf** [comment]
I'll be sure to take a look at this tomorrow! 😊

**Comment by salemlf** [comment]
This looks great, thanks sm for the fix! ☺️

**Review by salemlf — APPROVED** [review]

**Comment by jyo142** [comment]
@salemlf Love the app! It is very beautiful as well :) Would love to keep contributing if you have any other tasks that need to be worked on in the future!

**Comment by salemlf** [comment]
> @salemlf Love the app! It is very beautiful as well :) Would love to keep contributing if you have any other tasks that need to be worked on in the future!

That'd be great! Gonna start creating more issues for bugs/feature requests so it's easier to find places where people can contribute!

**Pull request merged**